assets/css/style.css

```diff
@@ -0,0 +1,134 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #d9dcd6;
+}
+
+/*Nav bar*/
+.header {
+  padding: 20px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+    "Lucida Sans", Arial, sans-serif;
+  background-color: #2a607c;
+  color: #ffffff;
+}
+
+.header h1 {
+  display: inline-block;
+  font-size: 48px;
+}
+
+.header h1 .seo {
+  color: #d9dcd6;
+}
+
+nav {
+  padding-top: 15px;
+  margin-right: 20px;
+  float: right;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  font-size: 20px;
+}
+
+header nav ul {
+  list-style-type: none;
+}
+
+header nav ul li {
+  display: inline-block;
+  margin-left: 25px;
+}
+
+nav a {
+  color: #ffffff;
+  text-decoration: none;
+}
+/*Background image*/
+.hero {
+  height: 800px;
+  width: 100%;
+  margin-bottom: 25px;
+  background-image: url("../images/digital-marketing-meeting.jpg");
+  background-size: cover;
+  background-position: center;
+}
+
```

Comment:
Whilst this does work, you can't add an `alt` tag to a CSS image which was a requirement for this homework. I'd consider moving this back to the index.html so you can add this.

Reply:
I think there is a slight confusion here @LeeFarnell2U. The image was always in the .css file and was given this way for this project. In this case the image is purely there for presentation reasons and is fine as a background image and hence no need for an `alt` attribute :)